includes/discovery/sensors/current/mgeups.inc.php

```diff
@@ -3,7 +3,7 @@
 // MGE UPS
 if ($device['os'] == 'mgeups') {
     echo 'MGE ';
-    $oids = trim(snmp_walk($device, '1.3.6.1.4.1.705.1.7.1', '-OsqnU'));
+    $oids = trim(snmp_walk($device, '1.3.6.1.4.1.705.1.7.2.1.5', '-OsqnU'));
```

Comment:
The OID change is because on Galaxy/Evolution 1.3.6.1.4.1.705.1.7.1.1/2/3/etc are the number of phases. On the Comit 1.3.6.1.4.1.705.1.7.1 equal to 3 (the number of phases). On all three 1.3.6.1.4.1.705.1.7.2.1.5.1/2/3/etc is equal to the value current of that phase. If only 1.3.6.1.4.1.705.1.7.2.1.5.1 exists, then it's a single phase system.